2025-03-04 19:29:04,295 DEBUG    root            Loaded Command Group: ['gcloud', 'run']
2025-03-04 19:29:05,677 DEBUG    root            Loaded Command Group: ['gcloud', 'run', 'services']
2025-03-04 19:29:07,021 DEBUG    root            Loaded Command Group: ['gcloud', 'run', 'services', 'update']
2025-03-04 19:29:07,039 DEBUG    root            Running [gcloud.run.services.update] with arguments: [--min-instances: "<googlecloudsdk.command_lib.run.flags.ScaleValue object at 0x796eae7677d0>", --region: "africa-south1", SERVICE: "streamlit-app"]
2025-03-04 19:29:07,041 DEBUG    root            (gcloud.run.services.update) The [project] resource is not properly specified. Please specify the argument [--project] on the command line or set the property [core/project].
Traceback (most recent call last):
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 981, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 934, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/surface/run/services/update.py", line 196, in Run
    service_ref = args.CONCEPTS.service.Parse()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/concepts/handlers.py", line 90, in Parse
    return self.parse(self.arg_getter())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/command_lib/util/concepts/info_holders.py", line 397, in Parse
    result = self.concept_spec.Parse(self.attribute_to_args_map,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/concepts/concepts.py", line 464, in Parse
    return self._ParseFromValue(
           ^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/concepts/concepts.py", line 545, in _ParseFromValue
    return self.Initialize(
           ^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/concepts/concepts.py", line 424, in Initialize
    return self._resources.Parse(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/core/resources.py", line 1221, in Parse
    return self.ParseResourceId(collection, line, params or {},
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/core/resources.py", line 973, in ParseResourceId
    return parser.ParseResourceId(resource_id, kwargs, base_url, subcollection,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/core/resources.py", line 314, in ParseResourceId
    param_values[param] = resolver() if callable(resolver) else resolver
                          ^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/concepts/concepts.py", line 417, in <lambda>
    return lambda: deps_lib.Get(name, fallthroughs_map, parsed_args)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/concepts/deps.py", line 356, in Get
    return GetFromFallthroughs(
           ^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/concepts/deps.py", line 381, in GetFromFallthroughs
    return fallthrough.GetValue(parsed_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/concepts/deps.py", line 90, in GetValue
    value = self._Call(parsed_args)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/google-cloud-sdk/lib/googlecloudsdk/command_lib/run/resource_args.py", line 183, in _Call
    raise exceptions.ArgumentError(
googlecloudsdk.command_lib.run.exceptions.ArgumentError: The [project] resource is not properly specified. Please specify the argument [--project] on the command line or set the property [core/project].
2025-03-04 19:29:07,131 ERROR    root            (gcloud.run.services.update) The [project] resource is not properly specified. Please specify the argument [--project] on the command line or set the property [core/project].
